<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>id / pw</string>
  </property>
  <widget class="QCheckBox" name="idCheckbox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>아이디 찾기</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="pwCheckbox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>비밀번호 찾기</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
